tests/rustdoc-ui/doctest/stdout-and-stderr.stdout

```diff
@@ -0,0 +1,68 @@
+
+running 3 tests
+test $DIR/stdout-and-stderr.rs - (line 12) ... FAILED
+test $DIR/stdout-and-stderr.rs - (line 16) ... FAILED
+test $DIR/stdout-and-stderr.rs - (line 7) ... FAILED
+
+failures:
+
+---- $DIR/stdout-and-stderr.rs - (line 12) stdout ----
+Test executable failed (exit status: 101).
+
+stderr:
+
+thread 'main' panicked at /tmp/rustdoctestSI6fSq/doctest_bundle_2024.rs:6:1:
```

Comment:
Is the `rustdoctestSI6fSq` path stable, i.e. won't it be different everytime the test runs?

Reply:
That's a `tempdir` somewhere, cf. #138475

Reply:
Yeye, needs norm

Reply:
Arf, I keep forgetting this one. ><